src/Groupper.ts

```diff
@@ -431,7 +439,15 @@ export class GroupperAPI implements Types.GroupperAPI {
         // Making sure groupper's onFocus is called before modalizer's onFocus.
         this._tabster.focusedElement.subscribeFirst(this._onFocus);
 
-        win.document.addEventListener("mousedown", this._onMouseDown, true);
+        const doc = win.document;
+
+        const activeElement = dom.getActiveElement(doc);
+
+        if (activeElement) {
+            this._onFocus(activeElement as HTMLElement);
+        }
```

Comment:
why was this added?

Reply:
During the testing I've found that we might have issues when the groupper is created when the focus is somewhere inside. So, I've added it here to make sure the current groupper is set accordingly.